components/sup/src/ctl_gateway/server.rs

```diff
@@ -141,91 +142,63 @@ impl CtlCommand {
 
 /// Server's client representation. Each new connection will allocate a new Client.
 struct Client {
-    handle: reactor::Handle,
-    state:  Rc<RefCell<SrvState>>,
+    state: Arc<Mutex<SrvState>>,
```

Comment:
Oh, did we finally get this code off of its own single-thread reactor? If so, :tada: